README.md

```diff
@@ -132,6 +132,7 @@ documentation for each plugin for configurable attributes.
 * `hddtemp` (see [collectd::plugin::hddtemp](#class-collectdpluginhddtemp) below)
 * `hugepages` (see [collectd::plugin::hugepages](#class-collectdpluginhugepages) below)
 * `intel_pmu` (see [collectd::plugin::intel_pmu](#class-collectdpluginintel_pmu) below)
+* `mcelog` (see [collectd::plugin::mcelog](#class-collectdpluginmcelog) below)
```

Comment:
can you please move this down to keep the list sorted alphabetical?

Reply:
Moved it down.